Readme.md

```diff
@@ -3,8 +3,36 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T519278)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# DropDownBox - How to filter data of a nested widget when a user types text into the editor
+# DropDownBox for DevExtreme - How to filter data of a nested DataGrid
 
-In this example, dxDropDownBox enables end-users to filter list items dynamically based on the text typed into the editor's input box on the client side.
+This example demonstrates how to filter data of a DropDownBox nested DataGrid component.
+
+![DropDownBox filtering](./dropdownbox-filtering.gif)
+
+Use [onInput](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#onInput), [onOpened](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#onOpened) and [onClosed](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#onClosed) event handlers to manipulate and display data.
```

Comment:
Do not use `manipulate` because it has a negative meaning

Reply:
I will change it for 'filter', cause it's more accurate.

Reply:
Ok